pydantic_extra_types/types/color.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, value: ColorType) -> None:
     def __get_pydantic_json_schema__(
         cls, core_schema: core_schema.CoreSchema, handler: GetJsonSchemaHandler
     ) -> JsonSchemaValue:
-        field_schema = {}  # type: ignore
+        field_schema: Dict[str, str] = {}
```

Comment:
To skip issues of mypy

Reply:
Pydantic v2 [uses](https://github.com/pydantic/pydantic/blob/main/pydantic/fields.py) PEP 585 for typing builtin collections, like `dict[str, str]`. Why not write it in a modern style, too?